server/instance_cloud_oauth.go

```diff
@@ -218,11 +218,11 @@ func (ci *cloudOAuthInstance) GetJiraBaseURL() string {
 }
 
 func (ci *cloudOAuthInstance) GetManageAppsURL() string {
-	return fmt.Sprintf("%s/plugins/servlet/applinks/listApplicationLinks", ci.GetURL())
+	return fmt.Sprintf("%s/plugins/servlet/applinks/listApplicationLinks", ci.GetJiraBaseURL())
```

Comment:
@raghavaggarwal2308 Are there any other similar changes that may need to be made throughout the plugin?

Reply:
@mickmister Nothing I observed as of now.